### PR TITLE
LPS-147476 Update Congrats Information Fragment to Mobile and Tablet View

### PR DESCRIPTION
**Pull request by luisxx123**

### CODE REVIEW

**Comment by luisxx123** [comment]
ci:test :sf

**Comment by silasbsi** [comment]
ci:test:stop

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.css [hunk 1]
```diff
@@ -82,3 +81,61 @@
...
+.outter-container {
+	display: none;
+}
+
+@media (max-width: 767.98px) {
+	.congrats-information {
+		width: 100%;
+	}
+
+	.right-column .download-print-share {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.css:98]: Change this selector to something like:
```suggestion
	.congrats-information .right-column .download-print-share {
```

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.css [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
+	height: fit-content;
 	padding: 40px 45px;
 	width: 749px;
 }
 
 .congrats-information h1 {
 	color: var(--color-neutral-black);
-	font-size: 32px;
 	font-weight: var(--font-weight-bold);
+	font-size: 32px;
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.css:13]: Reorder this css properties, please.

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.css [hunk 3]
```diff
@@ -82,3 +81,61 @@
...
+	.right-column .download-print-share {
+		display: none;
+	}
+
+	.right-column .congrats-info-price {
+		text-align: left !important;
+		padding-top: 2rem;
+	}
+
+	.congrats-information .row {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.css:107]: Reorder this css properties, please.

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.css [hunk 4]
```diff
@@ -82,3 +81,61 @@
...
+		max-width: 200px;
+	}
+
+	.outter-container {
+		display: flex;
+		justify-content: center;
+		width: 100%;
+	}
+
+	.outter-container ul {
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.css:136]: Reorder this css properties, please.

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.html [hunk 5]
```diff
@@ -1,42 +1,77 @@
-<div class="congrats-information container">
-	<div class="hide">
-		<img id="check" src="[resources:blue_check.svg]" />
+<div class="congrats-information rounded-lg container">
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.html:1]: Reorder these classes.

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.html [hunk 6]
```diff
@@ -1,42 +1,77 @@
...
+						</li>
+					</ul>
+				</div>
+			</div>
+		</div>
 	</div>
-</div>
\ No newline at end of file
+</div>
+<div class="container outter-container my-5">
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.html:62]: Reorder these classes.

File: modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.html [hunk 7]
```diff
@@ -1,42 +1,77 @@
...
+	<ul class="d-flex justify-content-around w-100 px-3">
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.html:63]: Reorder these classes.

**Review by silasbsi — CHANGES_REQUESTED** [review]
@luisxx123 there are some changes for you to make.

**Comment by luisxx123** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**